examples/rpmsgsocket/rpsock_server.c

```diff
@@ -315,7 +341,6 @@ static int rpsock_dgram_server(int argc, char *argv[])
 
   args->fd       = fd;
   args->nonblock = nonblock;
-  args->skippoll = true;
```

Comment:
Is `skippoll` still used? Maybe we can remove it?

Reply:
@pkarashchenko It's already removed here: https://github.com/apache/incubator-nuttx-apps/pull/1073/files#diff-ecc3b29b961b716dc9d3a3108645bfc63f186d1cbe9d1acefcb7fbe21716b28fL46.

Reply:
Yeah. I see now. Sometimes reviewing from a mobile device is tricky :)